docker-compose-services/postgres/README.md

```diff
@@ -52,6 +52,52 @@ hooks:
 There are also another non-plain-text formats that `pg_dump` can generate, and you might need to work with them. If that's the case, there is also
 a `ddev pg_restore` command that will restore `.ddev/pgsql-db/postgresql.db.dump` into `db`.
 
+## Multiple databases
+
+* By default, DDEV adds a single database called `db`. But it's possible to create additional databases when you first initialize this service.
+
+* The offical way to add multiple databases is outlined on the [docker image](https://hub.docker.com/_/postgres) under the "Initialization scripts"
+
+EG. Lets create a second database called `testing`.
+
+* Add `./pgsql-db:/docker-entrypoint-initdb.d` to the `volumes` section in to `./.ddev/docker-composer.postgres.yaml`
+
+  ```yaml
+      volumes:
+        - "./pgsql-db:/docker-entrypoint-initdb.d"
+  ```
+
+* Create `./.ddev/pgsql-db` folder if it doesn't already exist
+
+  ```bash
+  mkdir ./.ddev/pgsql-db -p
+  ```
+
+* Create a file called `./.ddev/pgsql-db/create_testing_database.sql` and add the following:
+
+  ```sql
+  CREATE DATABASE testing
+  ```
+
+* Note: All *.sql,*.sql.gz, & *.sh files in `./.ddev/pgsql-db` will be executed so you can create multiple database or even seed data.
+
+* Note: These files "are only run if you start the container with a data directory that is empty". IE. The first time you start the project. For exisiting projects, you must remove the volume first. Remember to backup the database first (Eg. `ddev pgsql_export`).
+
+### Removing a postgres volume
+
+  ```bash
+  # Display all docker volumes.
+  docker volume ls
+
+  # Stop DDEV.
+  ddev stop
+
+  # Remove 'example' project postgres volume.
+  docker volume rm ddev-example_postgres
```

Comment:
ddev automatically removes volumes that are properly named with `ddev delete`, so I don't think this set of instructions is needed.

Reply:
You're right. `DDEV delete` is more elegant solution.  I feel nervous when "an existing project" and "delete" are used so close together.


Side note:
`DDEV delete` generates an error because it can't generate a snapshot.
```
Failed to remove project test-psql: 
unable to snapshot database, 
```

This project has the mysql database services disables it doesn't I don't need. Running `ddev stop --omit-snapshot` works, although I get a warning ...
```
Unable to delete drud/ddev-dbserver-mariadb-10.3:v1.18.0-test-psql-built: no such image
```
Not sure if DDEV should be trying to delete a service if it's been disabled or my config file still has some distant reference.